lib/jsdom/browser/Window.js

```diff
@@ -659,6 +660,7 @@ function stopAllTimers(timers) {
     const timer = timers[key];
     // Need to .call() with undefined to ensure the thisArg is not timer itself
     timer[1].call(undefined, timer[0]);
+    delete timers[key];
   });
```

Comment:
Would just setting `timers.length = 0` after all this make sense?

Reply:
timers is a dict, so I don't think that'd work.

Reply:
Oh, but we're resetting them to empty in callers already, so this is unnecessary.